src/AccessControl/Owned.py

```diff
@@ -13,21 +13,10 @@
 """Support for owned objects
 """
 
-from zope.deferredimport import deprecated
-
 # BBB
 from AccessControl.owner import EditUnowned
 from AccessControl.owner import EmergencyUserCannotOwn
 from AccessControl.owner import UnownableOwner
 from AccessControl.owner import absattr
 from AccessControl.owner import ownableFilter
 from AccessControl.owner import ownerInfo
```

Comment:
That's also obsolete. I'd add a deprecation for version 6 here. I don't know if it's safe to remove.